.github/ISSUE_TEMPLATE/contribute-to-developer-blog.yml

```
@@ -0,0 +1,44 @@
+name: Contribute to the dbt Developer Blog
+description: >
+  For proposing a new post on the dbt Developer Blog.
+labels: ["content","developer blog"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We're excited to hear about your idea for the dbt Developer Blog. This template will help lay out the proposed post and then we will work with on next steps!
+
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you?
+      placeholder: your preferred email and/or dbt Slack handle
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: read-contribution
+    attributes:
+      label: I have read the dbt Developer Blog contribution guidelines.
+      description: You can the contribution guide [here](../../contributing/developer-blog.md)
```

```suggestion
      description: You can find the contribution guide [here](../../contributing/developer-blog.md)
```
Missing the word 'find'